firebase.json

```diff
@@ -15,16 +15,42 @@
         "source": "/api/**",
         "function": "api"
       },
+      {
+        "source": "/flows",
+        "function": "api"
+      },
       {
         "source": "**",
         "function": "dynamicIndex"
       }
-    ]
+    ],
```

Comment:
I wasn't expecting to this added as a new thing, I assume GitHub or Git is acting weird.